### PR TITLE
Make anne-bonny and bonnyci[bot] equivalent

### PR DESCRIPTION
**Pull request by jamielennox**
In the BonnyCI migration to using integrations the role of anne-bonny
will hereafter be played by bonnyci[bot]. To ease this transition treat
any reference in config or status to anne-bonny as equivalent to
bonnyci[bot].

This is not a long term good idea, but will let us do the transition and
revert this patch in a week or so.

Change-Id: I9b144c2b978eb82fcee42c08c60cde8142abd02a
Signed-off-by: Jamie Lennox <jamielennox@gmail.com>

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/zuul/39/1491863276.28/bonnyci-run-check : FAILURE in 3m 10s

File: zuul/trigger/github.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ def toList(item):
 
         efilters = []
         for trigger in toList(trigger_config):
+            statuses = [s.replace('anne-bonny', 'bonnyci[bot]'
```
Comment (omgjlk) [inline zuul/trigger/github.py]: Missing a closing paren on the `replace()` call.

**Review by omgjlk — COMMENTED** [review]
I like the intent here, and I agree that it'll be the smoothest transition. It'll require two restarts, but those are easier to handle than coordinating a flag day of repo configuration changes.

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/zuul/39/1491864354.85/bonnyci-run-check : FAILURE in 3m 10s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/zuul/39/1491864777.01/bonnyci-run-check : FAILURE in 3m 08s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/zuul/39/1492471460.42/bonnyci-run-check : FAILURE in 3m 10s

**Review by omgjlk — APPROVED** [review]

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/gate_github/BonnyCI/zuul/39/1492473106.38/bonnyci-run-gate : FAILURE in 35m 46s

**Comment by jamielennox** [comment]
reverify

**Comment by jamielennox** [comment]
gate-recheck

**Pull request merged**